apps/angular/src/app/features/anime-filter-form/anime-filter-form.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="filter-form-wrapper">
+	@if (animeFilterFormGroup) {
+		<form
+			class="filter-form"
+			[formGroup]="animeFilterFormGroup">
+			<mat-form-field>
+				<mat-label>Types</mat-label>
+				<mat-select
+					multiple
+					[formControl]="animeFilterFormGroup.controls.types">
+					@for (type of animeTypes; track type) {
+						<mat-option [value]="type">{{type}}</mat-option>
+					}
+				</mat-select>
+			</mat-form-field>
+				<mat-form-field
+					class="search search-form-field"
+					appearance="outline">
+					<mat-label>Anime title</mat-label>
+					<input
+						matInput
+						placeholder="Type to Search"
+						class="search-form__input"
+						type="search"
+						[formControl]="animeFilterFormGroup.controls.search"/>
```

Comment:
We usually place closing part on a new line to make it more clear where the list of attributes ends
```suggestion
					<input
						matInput
						placeholder="Type to Search"
						class="search-form__input"
						type="search"
						[formControl]="animeFilterFormGroup.controls.search"
				  />
```